quant docs: add and clean up BatchNorm{n}d

vkuzo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #40377 quant docs: add and clean up ELU
* **#40346 quant docs: add and clean up BatchNorm{n}d**
* #40345 quant docs: add and clean up InstanceNorm{n}d
* #40343 quant docs: add and clean up GroupNorm
* #40342 quant docs: add and clean up LayerNorm
* #40341 quant docs: add and clean up hardtanh
* #40340 quant docs: add and clean up hardsigmoid
* #40323 quant docs: clean up hardswish

Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

Differential Revision: [D22152633](https://our.internmc.facebook.com/intern/diff/D22152633)

vkuzo committed:
quant docs: add and clean up BatchNorm{n}d

Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

[ghstack-poisoned]

vkuzo committed:
Update on "quant docs: add and clean up BatchNorm{n}d"


Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

Differential Revision: [D22152633](https://our.internmc.facebook.com/intern/diff/D22152633)

[ghstack-poisoned]

supriyar reviewed (approved):
Thanks for helping out with this!

vkuzo committed:
Update on "quant docs: add and clean up BatchNorm{n}d"


Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

Differential Revision: [D22152633](https://our.internmc.facebook.com/intern/diff/D22152633)

[ghstack-poisoned]

vkuzo committed:
Update on "quant docs: add and clean up BatchNorm{n}d"


Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

Differential Revision: [D22152633](https://our.internmc.facebook.com/intern/diff/D22152633)

[ghstack-poisoned]

vkuzo committed:
Update on "quant docs: add and clean up BatchNorm{n}d"


Cleans up docstrings for quantized BatchNorm and adds to quantization docs

Test plan:
* build on Mac OS and inspect

Differential Revision: [D22152633](https://our.internmc.facebook.com/intern/diff/D22152633)

[ghstack-poisoned]
